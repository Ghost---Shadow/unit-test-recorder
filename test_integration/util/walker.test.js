const {
  walk,
} = require('../../src/util/walker');

describe('walker', () => {
  describe('walk', () => {
    it('should return list of all files in directory', () => {
      const allFiles = walk('./test_integration/util/walking_test');
      expect(allFiles).toMatchSnapshot();
    });
  });
});
