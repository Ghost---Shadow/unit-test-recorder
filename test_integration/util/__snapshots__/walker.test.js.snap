// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`walker walk should return list of all files in directory 1`] = `
Array [
  "test_integration\\\\util\\\\walking_test\\\\a.js",
  "test_integration\\\\util\\\\walking_test\\\\directory\\\\b.jsx",
  "test_integration\\\\util\\\\walking_test\\\\directory\\\\e.tsx",
  "test_integration\\\\util\\\\walking_test\\\\f.ts",
]
`;
